PHPfiles/ProductCalendar.php

```diff
@@ -38,12 +50,15 @@ function renderCalendar() {
                 $columnNumber = 0;
             }
             if ($productId === ProductCalendar::NoLunchProductId) {
-                $dateString = date_format(new DateTime($productDate), "M d, Y");
-                HtmlHelpers::writeTableCell("$dateString <p><i>No lunch</i></p>", ThemeConstants::TableCellNothingToOrderStyle);
+                ProductCalendar::renderNoLunchTableCell($productDate);
             } elseif ($productId === ProductCalendar::NoDetailsProductId) {
```

Comment:
What does this line mean? Is this when it's a blank cell because it's not part of the given month?

Reply:
Correct